src/app/api/ai-plugin/route.ts

```diff
@@ -163,6 +163,17 @@ export async function GET() {
                 "The amount of tokens to sell before fees, represented as a decimal string in token units. Not Atoms.",
             },
           ],
+          // requestBody: {
```

Comment:
The Bitte Wallet Agent Requests seem to be reading only GET request parameters and populating the body with the same fields. The current API spec (as defined here) breaks the Swagger UI - rendering it unusuable for POST requests. See here:

https://near-cow-agent.vercel.app/

cc @Markeljan